### DISCUSSION TITLE
Responses are truncated

### DISCUSSION THREAD

**Discussion by vishnuc** [community/community#31781] [Copilot Conversations]
Hi , I just subscribed to copilot today and trying it on vscode

Several times results are truncated if I use cntrl+enter , like below.

```
//fetch url and return body in http server
package main

import (
	"

======= 
```

and sometimes codes are fully commented

```

//fetch url and return body in http server
// package main
// 
// import (
// 	"fmt"
// 	"io"
// 	"net/http"
// 	"os"
// )

=======
```

Is there any option to increase response to show full result. , thanks.

**Comment by PythonCoderAS** [comment]
Sometimes it only generates a small fragment. You can always open the GitHub Copilot menu to get other suggestions.

**Comment by DenverCoder1** [comment]
Try typing a few extra letters and it will finish the suggestion.

For the commented code issue, if you were to type the letter `p` or a few of the letters from `package`, the rest would get filled out uncommented.

Note that GitHub Copilot tries not to suggest email addresses, websites, or personal information, at least when you have not supplied it yourself, so if it stops after a open quote, for example, sometimes this is the reason. The only solution to this is just to type out that part yourself, and it should continue suggesting normally after that.

While Copilot gives a significant productivity increase, don't expect it to be perfect or do everything for you.
